dig_test.go

```diff
@@ -2455,214 +2456,266 @@ func TestFailingFunctionDoesNotCreateInvalidState(t *testing.T) {
 	}), "second invoke must fail")
 }
 
-func TestDotGraph(t *testing.T) {
-	type T1 struct{}
-	type T2 struct{}
-	type T3 struct{}
-	type T4 struct{}
+func compareCtor(t *testing.T, expected *dot.Ctor, ctor *dot.Ctor) {
```

Comment:
nit: rename to `assertCtorEquals` or similar? "compare" suggests that this will compare them without failing the test.